### DISCUSSION TITLE
Wiki.js footer overlapping the last line on the preview panel

### DISCUSSION THREAD

**Discussion by Zuquim** [requarks/wiki#3767] [Error / Bug Report] [answered]
**Wiki.js footer overlapping the last line on the preview panel**
When editing a long (enough to enable the preview panel scrolling) Markdown page (new or existing), the last line is almost completely hidden by Wiki.js' footer.

I don't know if it is common or even allowed to do it, but here's a screenshot of the issue: https://i.imgur.com/vbP9T3M.png

**Client Info**
- OS: MacOS 10.15.7
- Browser: Mozilla Firefox 86.0.1
- Browser: Mozilla Firefox Developer Edition 87.0

**Host Info**
- OS: Docker Container (official docker-compose.yml)
- Wiki.js version: 2.5.191
- Database engine: Postgres 11.11
- Node.js: 14.16.0

**Comment by Igorgro** [comment, marked as answer]
Duplicate of https://github.com/Requarks/wiki/discussions/3606
